docs/services/eventhub/eventhub.md

```diff
@@ -31,6 +31,228 @@ As an output of deployment, the operator stores a JSON formatted secret with the
 
 You can follow the steps [here](/docs/howto/resourceprovision.md) to deploy, view and delete resources.
 
-<!-- ## How would you use the Event Hubs Operator to support a real application?
+## Sample application
 
-TODO: Demo app -->
+You can use this example to better understand how to use the Event Hubs Operator in a real application. We'll deploy a pair of producer and consumer apps to Kubernetes that will send and receive messages from Event Hubs respectively. Both the apps are written in [Go](https://golang.org/) using the [Sarama library for Kafka](https://github.com/Shopify/sarama/).
+ 
+### Setup Azure Event Hubs
+
+Create an Azure Resource Group
+
+> Replace `location` if required
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: azure.microsoft.com/v1alpha1
+kind: ResourceGroup
+metadata:
+  name: eh-aso-rg
+spec:
+  location: southeastasia
+EOF
+```
+
+Use `kubectl get resourcegroups/eh-aso-rg` to confirm that the Resource Group was created (`PROVISIONED` status should show up as `true`).
+
+Create Event Hubs namespace
+
+> Replace `location` if required
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: azure.microsoft.com/v1alpha1
+kind: EventhubNamespace
+metadata:
+  name: eh-aso-ns
+spec:
+  location: southeastasia
+  resourceGroup: eh-aso-rg
+  sku:
+    name: Standard
+    tier: Standard
+    capacity: 1
+  properties:
+    kafkaEnabled: true
+    isAutoInflateEnabled: false
+    maximumThroughputUnits: 0
+EOF
+```
```

Comment:
This idiom of using `cat <<EOF` to feed the input directly into `kubectl` works well when the user will directly copy/paste the content from the web page, but when some editing is required (such as with `location` in this example), users will need to do some juggling to make changes.

Perhaps, instead use `cat <<EOF` to create a yaml file, then suggest to users that changes might be needed, then consume the file using `kubectl`.